packages/bierzo-wallet/src/routes/balance/index.e2e.spec.ts

```diff
@@ -63,9 +63,9 @@ withChainsDescribe("E2E > Balance route", () => {
     expect(username).toBe("Get your human readable");
   }, 45000);
 
-  it("should create username", async () => {
+  it("should create personalized username", async () => {
```

Comment:
"personalized username" -> "personalized address" or "username". Both terms are fine for me in code, but "personalized username" does not make sense